doc/sphinx/src/models.rst

```diff
@@ -3,3 +3,409 @@
++--------------------------+ Specific Heat        +------------------------+
+| :math:`C_V(\rho, E)`     | Capacity             | Density, Energy        |
++--------------------------+----------------------+------------------------+
+| :math:`B_S(\rho, T)`     | Isentropic Bulk      | Density, Temperature   |
++--------------------------+ Modulus              +------------------------+
+| :math:`B_S(\rho, E)`     |                      | Density, Energy        |
++--------------------------+----------------------+------------------------+
+| :math:`\Gamma(\rho, T)`  | Gruneisen Parameter  | Density, Temperature   |
++--------------------------+                      +------------------------+
+| :math:`\Gamma(\rho, E)`  |                      | Density, Energy        |
++--------------------------+----------------------+------------------------+
+
+A point of note is that "specific" implies that the quantity is intensive on a
+per unit mass basis.
+
+Units and conversions
+---------------------
+
+The default units for ``singularity-eos`` are cgs which results in the following
+units for thermodynamic quantities:
+
++------------+------------------+---------------------------------------+-----------------------+
+|Quantity    | Default Units    | cgµ conversion                        | Sesame Units          |
++============+==================+=======================================+=======================+
+|:math:`P`   | µbar             | 10\ :sup:`-12` Mbar                   | 10\ :sup:`-10` GPa    |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`\rho`| g/cm\ :sup:`3`   | 1                                     | 1                     |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`E`   | erg              | 10\ :sup:`-12` Mbar-cm\ :sup:`3`/g    | 10\ :sup:`-10` kJ/g   |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`T`   | K                | 1                                     | 1                     |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`C_V` | erg/g/K          | 10\ :sup:`-12` Mbar-cm\ :sup:`3`/g/K  | 10\ :sup:`-10` kJ/g/K |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`B_S` | µbar             | 10\ :sup:`-12` Mbar                   | 10\ :sup:`-10` GPa    |
++------------+------------------+---------------------------------------+-----------------------+
+
+Note: some temperatures are measured in eV for which the conversion is
+8.617333262e-05 eV/K.
+
+Implemented EOS models
+----------------------
+
+
+Ideal Gas
+`````````
+
+The ideal gas (aka perfect or gamma-law gas) model in ``singularity-eos`` takes
+the form
+
+.. math::
+
+    P = \Gamma \rho \hat{e}
```

Comment:
Is this just a me thing? I think of specific energy as \varepsilon, not \hat{e}. But I'm willing to be overruled. I don't think it matters much.

Reply:
As I mention in another comment, I think using `\Gamma` here is a little confusing since I think most hydro folks will expect to see
```
P = (\Gamma - 1)rho \varepsilon
```

Reply:
I'll switch everything to lowercase e as I agree that upper-case E is usually total energy.

Reply:
The ideal gas law was a mistake in the nomenclature that has been brought into consistency with the rest of the document